UI/data/locale/en-US.ini

```diff
@@ -1240,7 +1240,7 @@ YouTube.Actions.Notify.Title="YouTube"
 YouTube.Actions.Notify.CreatingBroadcast="Creating a new Live Broadcast, please wait..."
 
 YouTube.Actions.AutoStartStreamingWarning.Title="Manual start required"
-YouTube.Actions.AutoStartStreamingWarning="Auto start is disabled for this stream, click the \"GO LIVE\" button to publish your broadcast on YouTube."
+YouTube.Actions.AutoStartStreamingWarning="Auto-start is disabled for this event, click \"Go Live\". to start your broadcast"
```

Comment:
The period should be moved to the end of the string.

Reply:
Good catch